themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -138,6 +138,7 @@
               name="{$field.name}"
               title="{l s='At least 5 characters long' d='Shop.Forms.Help'}"
               type="password"
+              {if $field.autocomplete}autocomplete="{$field.autocomplete}"{/if}
```

Comment:
Should'nt we use Smarty `escape` function ?

And I know this is not the scope of this PR but maybe we should (in another PR) apply the `escape` to EVERY other value in this TPL 😅 

Wdyt @PierreRambaud ?

Reply:
Waiting for @PierreRambaud advice on this

Reply:
AFAIK, smarty already automatically escape html
`$smarty->escape_html = true;`
https://www.smarty.net/docs/en/variable.escape.html.tpl